extensions/DynamicPageList/DPLMain.php

```diff
@@ -2311,20 +2275,26 @@ public static function dynamicPageList( $input, $params, $parser, &$bReset, $cal
         }
 
         // SELECT ... FROM
-        if ($acceptOpenReferences)
-                 // SELECT ... FROM
-        	if (count($aImageContainer)>0) {
-	        	$sSqlSelectFrom = "SELECT $sSqlCalcFoundRows $sSqlDistinct " . $sSqlCl_to . 'ic.il_to, ' . $sSqlSelPage . "ic.il_to AS sortkey" . ' FROM ' . $sImageLinksTable . ' AS ic';
-        	}
-        	else {
-	        	$sSqlSelectFrom = "SELECT $sSqlCalcFoundRows $sSqlDistinct " . $sSqlCl_to . 'pl_namespace, pl_title' . $sSqlSelPage . $sSqlSortkey . ' FROM ' . $sPageLinksTable;
-        	}
-        else
+        if ($acceptOpenReferences) {
+			// SELECT ... FROM
+			if ( count( $aImageContainer ) > 0 ) {
+				$sSqlSelectFrom =
+					"SELECT $sSqlCalcFoundRows $sSqlDistinct " . $sSqlCl_to . 'ic.il_to, ' .
+					$sSqlSelPage . "ic.il_to AS sortkey" . ' FROM ' . $sImageLinksTable . ' AS ic';
+			} else {
+				$sSqlSelectFrom =
+					"SELECT $sSqlCalcFoundRows $sSqlDistinct " . $sSqlCl_to .
+					'pl_namespace, pl_title' . $sSqlSelPage . $sSqlSortkey . ' FROM ' .
+					$sPageLinksTable;
+			}
+		} else {
+        	$dplRevisionQueryTables = " {$dplTableSet->getTables()} ";
 			$sSqlSelectFrom = "SELECT $sSqlCalcFoundRows $sSqlDistinct " . $sSqlCl_to . $sPageTable.'.page_namespace AS page_namespace,'.
             					$sPageTable.'.page_title AS page_title,'.$sPageTable.'.page_id AS page_id' . $sSqlSelPage . $sSqlSortkey .
                                 $sSqlPage_size . $sSqlPage_touched . $sSqlRev_user .
                                 $sSqlRev_timestamp . $sSqlRev_id . $sSqlCats . $sSqlCl_timestamp .
-                                ' FROM ' . $sSqlRevisionTable . $sSqlCreationRevisionTable . $sSqlNoCreationRevisionTable . $sSqlChangeRevisionTable . $sSqlRCTable . $sSqlPageLinksTable . $sSqlExternalLinksTable . $sPageTable;
+                                ' FROM ' . $sSqlRevisionTable . $dplRevisionQueryTables . $sSqlRCTable .
+							  $sSqlPageLinksTable . $sSqlExternalLinksTable . $sPageTable;}
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this closing curly brace to the next line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1109)

[project-id]: wikia:app